src/modules/m_sqloper.cpp

```diff
@@ -19,46 +20,104 @@
 
 #include "inspircd.h"
 #include "modules/sql.h"
-#include "modules/hash.h"
 
-class OpMeQuery : public SQLQuery
+class OpGetQuery : public SQLQuery
 {
  public:
+	// This variable will store all the OPER blocks that are from the config file. This is so we are always aware of which OPER blocks aren't from this module.
```

Comment:
Can you add line breaks to your comments. Ideally they shouldn't be longer than the line of code associated with them.

Reply:
I fixed the issues with comments.